mempool/mempool_shared_proto/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "mempool_shared_proto"
```

Comment:
Could you change the name of the crate to use `-`'s. We're trying to standardize our crate names